common/components/ui/Modal.scss

```diff
@@ -66,6 +65,11 @@ $m-anim-speed: 400ms;
   animation: modal-open $m-anim-speed ease 1;
   text-align: left;
 
+  &.is-small {
+    width: calc(100% - 40px);
+    max-width: 850px;
+  }
```

Comment:
This still isn't what I meant for how the modal works. The idea from the original implementation is that the modal's _content_ is what defines the width, and it just shrinks and grows to that. So the idea is that the Footer's modal content would be wrapped in some element that is `width: 850px;`. Since the content is `display: flex`, it'll shrink it when it gets below that.

It was an unfamiliar way of doing things to me when it was first proposed, but it's actually pretty elegant since you don't have to add things like this.

Reply:
I'm more familiar with having set sizes for modals so they remain consistent across the site and adhere to design specs no matter the content. I'm fine with having the content define the modal, but main problem this css is trying to solve is responsiveness on mobile. With the previous implementation, it meant that you would get a skinny modal, about 50% the size of the screen (viewport width).

What about if I added a media query for mobile screens to adjust the width. Something like this

```
@media(max-width: 600px) {
    .modal {
          width: calc(100% - 40px);
     }
}
```

Reply:
That makes sense to me! Go ahead and add it. Just as an aside, you can put media queries inside of selectors to reduce repeated selectoring, like so:

```
.modal {
  width: auto;

  @media(max-width: 600px) {
    width: calc(100% - 40px);
  }
}
```